PoGo.NecroBot.CLI/Config/Translations/translation.ja-JP.json

```diff
@@ -78,7 +78,7 @@
     },
     {
       "Key": "eventFortUsed",
-      "Value": "名称: {0} 経験値: {1}, ジェム: {2}, アイテム: {3}, 経度: {4}, 緯度: {5}"
```

Comment:
This is the Japanese Translation for this, like all the others why are you translating them....